### PR TITLE
Add TRS space

### PR DESCRIPTION
**Pull request by Ultrabitch**
**Add space for TraderSquare**  
  
Etherscan: https://etherscan.io/token/0x1ef4d930097d30689b16945aa8286c62ce7f75df  
CoinGecko: https://www.coingecko.com/en/coins/tradersquare 
Website: https://www.trader-square.fr/team
Twitter: https://twitter.com/tradersquareoff

### CODE REVIEW

File: spaces/TraderSquare/index.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "TraderSquare",
+  "network": "1",
+  "symbol": "TRS",
+  "skin": "example",
```
Comment (gokaiorg) [inline spaces/TraderSquare/index.json:5]: If you use the default theme you will have to remove the skin field.

**Review by gokaiorg — CHANGES_REQUESTED** [review]
Space folder should be in lowercase `spaces/tradersquare/index.json`
Do not edit `example` space directory.

File: spaces/TraderSquare/index.json (hunk 1 above)
Comment (Ultrabitch) [reply spaces/TraderSquare/index.json:5]: Thank you for your help

Comment (Ultrabitch) [reply spaces/TraderSquare/index.json]: Ok that's it. I restored initial exemple files, I added new skin"tradersquare.scss" and I update "index.json" with corresponding skin.
Thanks

**Comment by bonustrack** [comment]
@Ultrabitch It's now possible to create and edit a space without doing pull requests but using ENS and IPFS instead. Could you use this new method to create your space? Here is the guide: https://docs.snapshot.page/guides/create-a-space (you can keep your PR for add the images and skin)

**Comment by Ultrabitch** [comment]
Hey,
I've made the requested changes here, could you please update this PR ?

**Comment by Ultrabitch** [comment]
@gokaiorg done !

**Comment by gokaiorg** [comment]
@Ultrabitch We have made a major update. You should now use ENS and IPFS method to create your space. Here is the guide: https://docs.snapshot.page/guides/create-a-space. I'm closing this PR for now, you will need to open a new one to add your space images. Thank you.

**Pull request closed without merging**